README.md

```diff
@@ -35,6 +35,8 @@ Pointing Træfik at your orchestrator should be the _only_ configuration step yo
 
 ---
 
+:construction: As stated in [our latest 1.7 release note](https://blog.containo.us/traefik-1-7-yet-another-slice-of-awesomeness-2a9c99737889#782d), a major overhaul is currently occuring on the [master](https://github.com/containous/traefik/tree/master) branch. It will remain in constant evolution and prone to change with little notice.
```

Comment:
"the 1.7 release note"
"is occurring" (two r and currently not necessary) - or replace by is in progress.
It: unclear antecedent (major overhaul or the branch). Maybe "This branch will ..."
Maybe add what it means for the user -> ... so use it for test purposes only.

Reply:
I'd also rather use "significant update" instead of "major overhaul"

Reply:
Fixed